src/System.Runtime/tests/ArrayTests.netcoreapp.cs

```diff
@@ -13,24 +13,36 @@ namespace System.Tests
     public static class ArrayNetcoreappTests
     {
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/corert/issues/3650 - Wrong exception thrown", TargetFrameworkMonikers.UapAot)]
         public static void CreateInstance_Type_Int_Invalid()
         {
-            // Type is not a valid RuntimeType
-            AssertExtensions.Throws<ArgumentException>("elementType", () => Array.CreateInstance(Helpers.NonRuntimeType(), 0));
+            foreach (Type nonRuntimeType in Helpers.NonRuntimeTypes)
+            {
+                // Type is not a valid RuntimeType
+                AssertExtensions.Throws<ArgumentException>("elementType", () => Array.CreateInstance(nonRuntimeType, 0));
+            }
```

Comment:
Nit: instead of foreach, you can use Assert.All, e.g.
```C#
Assert.All(Helpers.NonRuntimeTypes, nonRuntimeType => AssertExtensions.Throws<ArgumentException>("elementType", () => Array.CreateInstance(nonRuntimeType, 0)));
```
The benefit of that is, if any fails, rather than stopping immediately and not processing any of the subsequent elements, it'll process them all and show you how many and which failed out of the total.